.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build MultiSense ROS
+
+on:
+  push:
+    branches:
+      - master
+  release:
+  pull_request:
+    branches:
+      - master
```

Comment:
Do you want to add a workflow_dispatch trigger too?

Reply:
does that allow manual running of the workflow? Seems like it would be nice to have as an option